core/storage/suggestion/gae_models.py

```diff
@@ -615,61 +615,57 @@ def export_data(cls, user_id):
         return user_data
 
 
-class ReviewerAndSuggestionCountsModel(base_models.BaseModel):
-    """Records the total number of reviewers for each suggestion type and the
-    total number of suggestions in review for each suggestion type. There is
-    only ever one instance of this model with ID
-    REVIEWER_AND_SUGGESTION_COUNTS_MODEL_ID.
+class CommunityContributionStatsModel(base_models.BaseModel):
+    """Records the contributor dashboard contribution stats. This includes the
+    total number of reviewers for each suggestion type and the total number of
+    suggestions in review for each suggestion type. There is only ever one
+    instance of this model with ID COMMUNITY_CONTRIBUTION_STATS_MODEL_ID.
```

Comment:
```suggestion
    instance of this model, and its ID is ID COMMUNITY_CONTRIBUTION_STATS_MODEL_ID.
```

Important distinction: we want to establish that there's only one model, and it's ID is the shared ID. Otherwise, this wording implies there could be other models with different IDs.

Reply:
You're right - your wording is a bit more definitive, I changed it, thanks Ben!